test/bitcoincash.test.js

```diff
@@ -0,0 +1,37 @@
+/* global describe, it */
```

Comment:
Creating bitcoin cash raw transaction and broadcasting on network throws following error:
`16: mandatory-script-verify-flag-failed (Script failed an OP_EQUALVERIFY operation). Code:-26`
I am broadcasting through API call using URL `https://cashexplorer.bitcoin.com/api/tx/send`

Reply:
Show us the code you used to make the transaction. Thanks.

Reply:
Wrong pubkeyhash I would say.. Signature wasn't checked?

Reply:
@junderw 
code excerpt which works for BTC:
https://gist.github.com/skynxt/d395a115ae9bfbde6e9744b0e962beaa

code excerpt creating tx for BCH that shows error:
https://gist.github.com/skynxt/2e17f5ff088fc4f47d339d55aee14045

Another excerpt of BCH create tx which throws error too :
https://gist.github.com/skynxt/d395a115ae9bfbde6e9744b0e962beaa

@afk11 I am using correct private key for signing. Probably the above code might give you insight on how transactions are being created at my end.
Thank you.

Reply:
You’re signing the output value not the prevOut value.

You need to pass sign the input value (previous Output value) not the output value